### DISCUSSION TITLE
Background Artwork - Don't Downlod?

### DISCUSSION THREAD

**Discussion by jfreiman** [CodyBerenson/PGMA-Modernized#154] [General]
Is there a way to prevent the Agents from downloading the Background Artwork for Plex?  
I much prefer the snapshot's that PMS grabs and use that as the Background Artwork -- much of the time the Backcover art is low quality and/or if there is text on the back cover it really looks messy when displayed on a big screen TV.

**Comment by JPH71** [comment]
This has been implemented....
It will be a new preference...
nearly finished with IAFD
